internal/indexer/definitions/milkie.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: Milkie
+name: Milkie
+identifier: milkie
+description: Milkie is a private torrent tracker for GENERAL.
+language: en-us
+urls:
+  - https://www.milkie.cc
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: apikey

```

Comment:
Are you're sure it's an api key and not passkey? Doesn't matter more than it could be confusing for the user to get it if they differ.

Reply:
sure 100% I mean this is what is explicitly mentioned in the indexer page.
In the past it seems it was called Download key but is now showing apikey.